libs.autobuild

```diff
@@ -9,7 +9,10 @@ cmake_package 'knowledge_reasoning/factpp' do |pkg|
     pkg.depends_on 'base/cmake'
 end
 cmake_package 'knowledge_reasoning/owlapi'
-cmake_package 'tools/graph_analysis'
+cmake_package 'tools/graph_analysis' do |pkg|
+    bin,_,_ = Rock.activate_python_path(pkg)
+    pkg.define 'PYTHON_EXECUTABLE', bin if bin
```

Comment:
Do you have a feature flag to decide whether the python bindings are built/installed at all ?

If you do, you also need to control it. Python might be installed on the machine, but the build decides that it should not built (think: production embedded build in CI where the worker needs Python but the target system does not)

If you do not, would be best to add one :P

As I said in my first comment, I'd be fine to merge it as-is. This is a suggestion.

Reply:
For Ruby we have BINDINGS_RUBY usage in base/cmake. If I understand you correctly you suggest the introduction of BINDINGS_PYTHON?

Reply:
That's it, yes.

Reply:
K, then I merge this as-is and update base/cmake